pkgs/development/libraries/simpleitk/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, cmake, git, swig, lua, itk }:
+
+stdenv.mkDerivation rec {
+  _name    = "simpleitk";
```

Comment:
The use of the underscore is non-standard. Instead, use either a `let` expression or arguments that are commonly used, like `pname` and `version`.